app/api/video/services_views.py

```diff
@@ -0,0 +1,31 @@
+import boto3
```

Comment:
I don't get this `services_views` name. why not just services?

Reply:
i showed that it refers to views and can be services_serializer and etc, i dont know if this is bad practic if that true i will rename

Reply:
I don't see value in such naming. I would not recommend naming like this.